### PR TITLE
Feature/ui hydrator app status cdap

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Adds a separate detailed view for custom app & etl app
- Fixes the route to detailed app in all features
- Fixes appropriate icon to be shown for appropriate apps

### CODE REVIEW

File: cdap-ui/app/features/apps/controllers/tabs/status-ctrl.js [hunk 1]
```diff
@@ -1,64 +1,61 @@
-/*
```
Comment (bdmogal) [inline cdap-ui/app/features/apps/controllers/tabs/status-ctrl.js]: copyright removed? is this intentional?

File: cdap-ui/app/features/apps/templates/detail.html [hunk 2]
```diff
@@ -53,13 +65,27 @@
     </div>
   </div>
   <div class="cdap-subnav-end"></div>
-  <ul class="nav nav-tabs slanted-tabs" role="tablist">
+  <ul class="nav nav-tabs slanted-tabs" role="tablist" ng-if="DetailController.isHydrator">
 
     <li role="presentation" ui-sref-active="active">
       <a ui-sref="apps.detail.overview.status" ui-sref-opts="{reload: true}" role="tab">Status</a>
     </li>
```
Comment (elfenheart) [inline cdap-ui/app/features/apps/templates/detail.html]: Is it possible to put this <li> block to be inside the <ul> from line 83-92, and simply put the ng-if isHydrator on the <li> itself?

File: cdap-ui/app/features/apps/templates/list.html [hunk 3]
```diff
@@ -90,7 +90,17 @@
...
         <td >
+          <a ui-sref="apps.detail.overview.programs({ appId: app.name })"
+              ng-if="app.artifact.name !== ListController.GLOBALS.etlBatch && app.artifact.name !== ListController.GLOBALS.etlRealtime"
+              ng-click="MyOrderings.appClicked(app.name)">
+            <strong tooltip="{{app.name}}"
+                    tooltip-enable="app.name.length > 35"
+                    tooltip-append-to-body="true">
+              {{ ::app.name | myEllipsis:35 }}
+            </strong>
+          </a>
```
Comment (elfenheart) [inline cdap-ui/app/features/apps/templates/list.html]: It seems like we are going to be doing the check whether it is ETL apps quite a bit. Should we have a helper function in a service to check whether it is ETL apps or not?

File: cdap-ui/app/features/overview/controllers/apps-section-ctrl.js [hunk 4]
```diff
@@ -27,6 +27,7 @@ angular.module(PKG.name + '.feature.overview')
     })
       .then(function(res) {
         this.apps = this.apps.concat(res);
+        console.info('apps: ', this.apps);
```
Comment (elfenheart) [inline cdap-ui/app/features/overview/controllers/apps-section-ctrl.js]: remove?

File: cdap-ui/app/features/apps/controllers/tabs/status-ctrl.js (hunk 1 above)
Comment (ajainarayanan) [inline cdap-ui/app/features/apps/controllers/tabs/status-ctrl.js]: No. Accidental. Putting it back in..

**Comment by elfenheart** [comment]
:+1: LGTM

**Pull request merged**